src/Sylius/Bundle/AddressingBundle/Form/Type/CountryChoiceType.php

```diff
@@ -41,18 +41,16 @@ public function configureOptions(OptionsResolver $resolver): void
             ->setDefaults([
                 'choice_filter' => null,
                 'choices' => function (Options $options): iterable {
-                    if (null === $options['enabled']) {
-                        $countries = $this->countryRepository->findAll();
-                    } else {
-                        $countries = $this->countryRepository->findBy(['enabled' => $options['enabled']]);
+                    if ($options['enabled'] === true) {
+                        return  $this->countryRepository->findBy(['enabled' => $options['enabled']]);
                     }
 
-                    return $countries;
+                    return $this->countryRepository->findAll();
                 },
                 'choice_value' => 'code',
                 'choice_label' => 'name',
                 'choice_translation_domain' => false,
-                'enabled' => true,
+                'enabled' => false,
```

Comment:
Revert this change and leave it as it was before not to change the default behaviour, because this `CountryChoiceType` can be used in many places. Instead of changing it here, set the proper value of this field in the appropriate form: https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/AddressingBundle/Form/Type/ZoneType.php#L104